lms/djangoapps/edraak_university/models.py

```diff
@@ -59,3 +58,15 @@ def cleanup_id(id):
 
     class Meta:
         unique_together = ('user', 'course_key',)
+
+class UniversityIDSetting(models.Model):
+    """
+    Class related to university ID settings for a course
+    """
+    course_key = CourseKeyField(max_length=255, db_index=True)
```

Comment:
Make it unique, there's no point from adding multiple entries to a single course.